attestation-agent/attester/src/tdx/rtmr.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2024 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// The actual rtmr event data handled in DCAP
+#[repr(C, packed)]
+pub struct TdxRtmrEvent {
+    /// Always 1
+    version: u32,
+
+    /// The RTMR that will be extended.
+    /// As defined in https://github.com/confidential-containers/td-shim/blob/main/doc/tdshim_spec.md#td-measurement
+    /// we will use RTMR 2 for guest application code and configuration.
```

Comment:
nit: with the `DEFAULT_PCR_INDEX` it maps to RTMR3.

Reply:
Thanks for this, will fix